leaf-ui/index.html

```diff
@@ -205,13 +205,13 @@
 		<script src="./js/backendComm.js"></script>
 
         <div class = "attribution">
-            <img src="./images/site-logo.PNG" style="width:25%;height:25%;float:left">
+            <img src="./images/site-logo.PNG" class = "image-format" style="width:25%;height:25%;float:left; z-index: 1">
```

Comment:
If the `image-format` class is 
```
.image-format { 
  z-index: 0; 
}
```
why do we need` z-index: 1` in this line? I think the class is being overwritten. 
<img width="144" alt="Screen Shot 2021-07-12 at 4 42 56 PM" src="https://user-images.githubusercontent.com/59470488/125353186-37db8200-e330-11eb-9562-d6077f0ec7b0.png">

Reply:
We can delete the` image-format` class if you're not planning on using it elsewhere or remove the `style` part from the HTML in lines 208 and 214 and add it to the css class.